app/src/main/AndroidManifest.xml

```diff
@@ -3,33 +3,69 @@
     package="corp.kairos.adamastor">
 
     <uses-permission android:name="android.permission.VIBRATE" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Why do we need external storage access?

Reply:
its for the google's API to work properly